maths/find_max_recursion.py

```diff
@@ -0,0 +1,26 @@
+# Divide and Conquer algorithm
+def find_max(nums, left, right):
+    """
+    find max value in list
+    :param nums: contains elements
+    :param left: index of first element
+    :param right: index of last element
+    :return: max in nums
+
+    >>find_max([1, 3, 5, 7, 9, 2, 4, 6, 8, 10], 0, 9) == 10
```

Comment:
greater than >, greater than >, greater than >, space

If not, it is not recognizing as a test.

Reply:
you mean like this `nums = [9, 7, 5, 3, 2, 1]` ?

Reply:
Nope.  I mean:
```python
  >>> find_max(...
# not...
  >>find_max(...
```

Reply:
So that your tests appear here https://travis-ci.org/TheAlgorithms/Python/builds/595103352#L382

Reply:
Thanks, I corrected follow as you say.

Reply:
Please review again! @cclauss